lisa/sut_orchestrator/azure/common.py

```diff
@@ -191,13 +203,15 @@ def __post_init__(self, *args: Any, **kwargs: Any) -> None:
                 "subscription_id",
                 "marketplace_raw",
                 "shared_gallery_raw",
-                "vhd",
+                "vhd_raw",
+                "vms_raw",
```

Comment:
Should it be removed?

Reply:
Resolved

Reply:
I mean the vms_raw should be removed. But vhd_raw is worth to filter out whitespaces. So it can be compatible with Azure pipelines.

Reply:
That is what I did. Please see new changes